packages/kbn-test/src/functional_tests/tasks.js

```diff
@@ -76,7 +76,7 @@ export async function startServers(options) {
       config,
       options: {
         ...opts,
-        extraKbnOpts: [...options.extraKbnOpts, '--dev'],
+        extraKbnOpts: [...options.extraKbnOpts, ...(options.installDir ? [] : ['--dev'])],
```

Comment:
Is there an explanation for this? I'm curious...

Reply:
The distributable doesn't support the `--dev` flag, it's only available when running from source.